### PR TITLE
Update readme with default theme

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 5064 .

#### Changes proposed in this Pull Request:
- Update readme with default theme colors
- 
- 

#### Screenshots / Screen Recording (if applicable)
-

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Comment by ektaghag-eaton** [comment]
The following screenshots are with all default colors (not use any prop such fontColor, backgroundColor, iconColor)

**Channel Value**

Screenshot 2023-12-27 at 4 57 20 PM

Screenshot 2023-12-27 at 4 57 34 PM

**Empty State**

Screenshot 2023-12-27 at 5 01 25 PM

Screenshot 2023-12-27 at 5 01 15 PM

**Header**
Screenshot 2023-12-27 at 5 03 30 PM

Screenshot 2023-12-27 at 5 03 45 PM

**Hero**
Screenshot 2023-12-27 at 5 04 50 PM

Screenshot 2023-12-27 at 5 04 39 PM

**InfolistItem** (Icon color is not as per design. Issue is already logged here BLUI-5068)
Screenshot 2023-12-27 at 5 05 47 PM
Screenshot 2023-12-27 at 5 05 57 PM

**Mobile Stepper**
Screenshot 2023-12-27 at 5 07 09 PM

Screenshot 2023-12-27 at 5 06 57 PM

**User Menu**
Screenshot 2023-12-27 at 5 08 15 PM
Screenshot 2023-12-27 at 5 08 25 PM

File: components/src/core/InfoListItem/InfoListItem.tsx [hunk 1]
```diff
@@ -194,7 +197,7 @@ export type InfoListItemProps = ViewProps & {
     /**
      * Color to use for text elements
      *
-     * Default: Theme.colors.text
+     * Default: theme.colors.text
```
Comment (huayunh) [inline components/src/core/InfoListItem/InfoListItem.tsx:200]: This color does not exist

File: components/src/core/ListItemTag/ListItemTag.tsx [hunk 2]
```diff
@@ -38,14 +38,14 @@ export type ListItemTagProps = TypographyProps & {
...
+     * Default: theme.colors.primary
      **/
     backgroundColor?: string;
 
     /**
      * Text color for the label.
      *
-     * Default: Theme.colors.onBackground for light background,
+     * Default: theme.colors.onBackground for light background,
      * or white[50] on dark background
```
Comment (huayunh) [inline components/src/core/ListItemTag/ListItemTag.tsx:49]: we no longer use `white[50]` anywhere

File: components/src/core/InfoListItem/InfoListItem.tsx [hunk 3]
```diff
@@ -165,7 +165,10 @@ export type InfoListItemProps = ViewProps & {
      */
     avatar?: boolean;
 
-    /** The color used for the background of the InfoListItem */
+    /** The color used for the background of the InfoListItem
+     *
+     * Default: 'transparent
```
Comment (huayunh) [inline components/src/core/InfoListItem/InfoListItem.tsx:170]: missing an `'`

File: docs/InfoListItem.md [hunk 4]
```diff
@@ -28,30 +28,30 @@ You can also supply an array of items that will be displayed as a character-sepa
...
+| divider           | Show a dividing line below the row                                       | `'full'` \| `'partial'`              | no       |                          |
+| fontColor         | Color to use for text elements                                           | `string`                             | no       |                          |
+| hidePadding       | Hide padding reserved for icons when there is no icon                    | `boolean`                            | no       | `false`                  |
+| iconAlign         | Icon alignment when `avatar` is set to`false`                            | `'left'` \| `'center'` \| `'right'`  | no       | `left`                   |
+| icon              | A component to render for the icon                                       | [`IconSource`](./Icons.md)           | no       |                          |
+| iconColor         | Color to use for the icon                                                | `string`                             | no       |                          |
```
Comment (huayunh) [inline docs/InfoListItem.md:42]: missing a default color value

File: docs/EmptyState.md [hunk 5]
```diff
@@ -20,15 +20,15 @@ import { EmptyState } from '@brightlayer-ui/react-native-components';
...
-| iconSize    | The size of the primary icon (100-200)         | `number`                      | no       | 100                           |
-| iconColor   | The color of the primary icon                  | `string`                      | no       | `theme.colors.outlineVariant` |
-| actions     | Additional components to render below the text | `JSX.Element`                 | no       |                               |
-| theme       | Theme value overrides                          | `$DeepPartial<ExtendedTheme>` | no       |                               |
+| Prop Name   | Description                                    | Type                          | Required | Default                         |
+| ----------- | ---------------------------------------------- | ----------------------------- | -------- | ------------------------------- |
+| title       | The primary text to display (first line)       | `string`                      | yes      | `theme.colors.onSurface`        |
+| description | The secondary text to display (second line)    | `string`                      | no       | `theme.colors.onSurfaceVariant` |
```
Comment (huayunh) [inline docs/EmptyState.md:26]: ???? These are not the default value for these two props

File: docs/UserMenu.md [hunk 6]
```diff
@@ -30,16 +30,16 @@ import { UserMenu } from '@brightlayer-ui/react-native-components';
...
-| menuItems       | An array of menu items to show in the bottom sheet       | `InfoListItemProps[]`         | yes      |         |
-| menuTitle       | Title to display in the bottom sheet                     | `string`                      | no       |         |
-| menuSubtitle    | Subtitle to display in the bottom sheet                  | `string`                      | no       |         |
-| theme           | Theme value overrides                                    | `$DeepPartial<ExtendedTheme>` | no       |         |
+| Prop Name       | Description                                              | Type                          | Required | Default                         |
+| --------------- | -------------------------------------------------------- | ----------------------------- | -------- | ------------------------------- |
+| avatar          | Avatar to display as the menu trigger                    | `JSX.Element`                 | yes      |                                 |
+| backgroundColor | Background color of the bottom sheet                     | `string`                      | no       |                                 |
```
Comment (huayunh) [inline docs/UserMenu.md:36]: No default background color here?

**Review by huayunh — CHANGES_REQUESTED** [review]
I assume the default colors in ListItemTag will be updated soon

File: docs/EmptyState.md (hunk 5 above)
Comment (ektaghag-eaton) [reply docs/EmptyState.md:26]: Referred description of this issue https://github.com/etn-ccis/blui-react-native-component-library/issues/475

**Comment by ektaghag-eaton** [comment]
> I assume the default colors in ListItemTag will be updated soon

Yes

File: docs/UserMenu.md (hunk 6 above)
Comment (ektaghag-eaton) [reply docs/UserMenu.md]: Updated backgroundColor  of bottomsheet as `theme.colors.surface`

File: docs/EmptyState.md (hunk 5 above)
Comment (ektaghag-eaton) [reply docs/EmptyState.md]: Removed it.

File: docs/InfoListItem.md (hunk 4 above)
Comment (ektaghag-eaton) [reply docs/InfoListItem.md]: Fixed...

File: components/src/core/ListItemTag/ListItemTag.tsx (hunk 2 above)
Comment (ektaghag-eaton) [reply components/src/core/ListItemTag/ListItemTag.tsx]: Removed

File: components/src/core/InfoListItem/InfoListItem.tsx (hunk 1 above)
Comment (ektaghag-eaton) [reply components/src/core/InfoListItem/InfoListItem.tsx]: Fixed.

**Comment by ektaghag-eaton** [comment]
As per our last discussion fixed InfolistItem and ListItemTag component

Screenshot 2023-12-28 at 11 08 55 AM
Screenshot 2023-12-28 at 11 09 06 AM

Screenshot 2023-12-28 at 11 09 27 AM
Screenshot 2023-12-28 at 11 09 15 AM

**Comment by ektaghag-eaton** [comment]
> As per our last discussion fixed InfolistItem and ListItemTag component
> 
> Screenshot 2023-12-28 at 11 08 55 AM Screenshot 2023-12-28 at 11 09 06 AM Screenshot 2023-12-28 at 11 09 27 AM Screenshot 2023-12-28 at 11 09 15 AM

@huayunh , please review.

File: components/src/core/ListItemTag/ListItemTag.tsx [hunk 7]
```diff
@@ -64,12 +62,8 @@ const listItemTagStyles = (
 }> =>
     StyleSheet.create({
         root: {
-            backgroundColor: props.backgroundColor || theme.colors.primary,
-            color:
-                props.fontColor ||
-                (Color(props.backgroundColor || theme.colors.primary).isLight()
-                    ? theme.colors.onNeutralShadedContainer
-                    : theme.colors.onPrimaryFilledContainer),
+            backgroundColor: props.backgroundColor || theme.colors.primaryFilledContainer,
```
Comment (ektaghag-eaton) [inline components/src/core/ListItemTag/ListItemTag.tsx:65]: @huayunh , updated backgroundColor and color prop.

**Review by huayunh — APPROVED** [review]

**Pull request merged**